### PR TITLE
🚚 Prefer the Term Preferences to Configuration

### PR DESCRIPTION
**Pull request by gilday**
When describing the mechanism by which the user tailors Pixeebot to their preferences, we want to use the term "preferences" instead of "config". Preferences implies that making changes is optional: Pixeebot still works out-of-the-box without requiring users to configure it. On the other hand, configuration is typically associated with a required step in the process of beginning to use a tool.

/close #work

### CODE REVIEW

**Comment by gilday** [comment]
I kept the URI path to the preferences page as `https://docs.pixee.ai/configuring/`, because I'm unsure the server that hosts our docs has the capability to redirect pages.

**Comment by gilday** [comment]
I stacked this change behind branch `iss-855/prepend-exclude-codemod-preferences`, because that change adds more to the `configuring.md` page, and I didn't want to get into a situation where I merge this change first and forget to use the term preferences in that outstanding PR.

**Review by sip49 — APPROVED** [review]
lgtm

File: docs/configuring.md [hunk 1]
```diff
@@ -2,36 +2,66 @@
...
-There are two approaches to configuring Pixeebot:
+Pixeebot works immediately after installation without any further configuration. Yet, Pixeebot exposes preferences for users to tailor its behavior when desirable.
 
-1. **Target repository configuration:**
-   Create a `pixeebot.yaml` file in the `.github` directory of the target repository. Configurations in the target repository will take precedence over other configurations.
+Users may set Pixeebot preferences at either the organization or repository levels.
 
-2. **Global repository configuration:**
-   Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global configuration that applies to multiple repositories.
+1. **Repository Preferences:**
```
Comment (terranosaurus) [inline docs/configuring.md:11]: preferences (lower case)

File: docs/configuring.md [hunk 2]
```diff
@@ -2,36 +2,66 @@
...
+   Create a `pixeebot.yaml` file in the `.github` directory of the target repository. Preferences in the target repository will take precedence over other preferences.
 
-## YAML
+2. **Organization-Wide preferences:**
```
Comment (terranosaurus) [inline docs/configuring.md:14]: -wide (lower case)

File: docs/configuring.md [hunk 3]
```diff
@@ -2,36 +2,66 @@
...
+   Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global preferences that applies to multiple repositories.
```
Comment (terranosaurus) [inline docs/configuring.md:15]: "This will serve as a global preference..." (singular)

Comment (terranosaurus) [reply docs/configuring.md:15]: Ok then I would say "This will serve as (the?) global preferences that apply to multiple repositories.

Comment (gilday) [reply docs/configuring.md:15]: How do you feel about replacing the "multiple repositories" with a more specific "all repositories in this installation"? I feel like "multiple repositories" is ambiguous. Which repositories? At Pixeebot's discretion?!

Comment (terranosaurus) [reply docs/configuring.md:15]: Yes I like that, we say something similar in the View/change preference modal in the UI.

**Pull request merged**